bot/controllers/memberAdd.js

```diff
@@ -0,0 +1,26 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
+class MemberAddController extends BaseController {
+  constructor(message) {
+    super(message);
+    const controller = this;
+    this.commands = [
+      new Command(
+        '!memberWelcome',
```

Comment:
"memberWelcome" doesn't really spell out what this command does. It can lead to ambiguity regarding its purpose. 
> Does this command welcome a new member?